### ISSUE TITLE
More task output windows

### ISSUE THREAD

**Issue by obrejla** [Microsoft/vscode#22156]
Hi!

- VSCode Version: 1.10.1
- OS Version: macOS Sierra

It would be absolutely great to have a possibility to allow VSCode to open New Task Output window for each triggered task. Why? It is a must have especially when running tests by Task. Because every newly triggered task clears the output of previous test run...and that's a pain if you don't remember all the test failures (no one does ;), because then you have to run all tests again. Uselessly, just because you triggered something after the first test run.

It could have the same ComboBox UI as more Terminal windows have.

(and use Terminal itself to run tests is not the solution, because such task can not be assigned to a keyboard shortcut, afaik)

Thanks for evaluation!

**Comment by dbaeumer** [comment]
@obrejla this will also come when tasks are executed in the terminal instead of the output panel. Here each task will reuse the terminal without clearing it. You will be able to even run two tasks in parallel. If you want to give it a try, please see the 1.9 and 1.10 release notes how to use tasks in terminal already today.

I will close the issue since this will come with running tasks in terminals. Please open a separate issue if you think how terminal are reused should be further polished.

**Comment by obrejla** [comment]
Running in terminal would be really great. Do you have some time estimate? Thanks!
